<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#lbl_logo{
image: url(:/resim/assets/images/logo-w.png);
}
#menu_frame, #top_frame, #bottom_frame, #logo_frame{
background-color: #162D6D;
}
#lbl_footer{
color: #EBEBEB;
font-size: 8pt;
font-style: italic;
padding-right: 20px;
}
#lbl_title{
color: #EBEBEB;
font-size: 20pt;
font-weight: bold;
}
#btn_users, #btn_employees, #btn_customers, #btn_vehicles, #btn_drivers, #btn_repairs, #btn_contracts,
#btn_routes, #btn_trips, #btn_attendance, #btn_payments, #btn_finance, #btn_constants, #btn_reports, #btn_settings{
color:#ffffff;
background-color: #162D6D;
border:1px solid#ffffff;
border-radius: 8px;
font-family: Century Gothic;
font-size: 12pt;
font-weight: bold;
text-align: left;
padding-left:5px;
}
#btn_users:hover, #btn_employees:hover, #btn_customers:hover, #btn_vehicles:hover, #btn_drivers:hover, #btn_repairs:hover, #btn_contracts:hover, #btn_routes:hover, #btn_trips:hover, #btn_attendance:hover, #btn_payments:hover, #btn_finance:hover, #btn_constants:hover, #btn_reports:hover, #btn_settings:hover{
color:#ffff00;
background-color: #162D6D;
border: 2px solid #ffff00;
border-radius: 8px;
font-family: Century Gothic;
font-size: 12pt;
font-weight: bold;
text-align: left;
padding-left:5px;
padding-top: 8px;
}
#btn_users:pressed, #btn_employees:pressed, #btn_customers:pressed, #btn_vehicles:pressed, #btn_drivers:pressed, #btn_repairs:pressed, #btn_contracts:pressed, #btn_routes:pressed, #btn_trips:pressed, #btn_attendance:pressed, #btn_payments:pressed, #btn_finance:pressed, #btn_constants:pressed, #btn_reports:pressed, #btn_settings:pressed{
color:#000000;
background-color: #f59724;
border: 2px solid #ffffff;
border-radius: 8px;
font-family: Century Gothic;
font-size: 12pt;
font-weight: bold;
text-align: left;
padding-left: 25px;
padding-top: 0px;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QFrame" name="top_frame">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lbl_title">
           <property name="font">
            <font>
             <family>Minalis Demo</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QFrame" name="bottom_frame">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lbl_footer">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>designed &amp; developed by Ayhan Ercan 2025</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="QFrame" name="menu_frame">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_logo">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QPushButton" name="btn_users">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>ClosedHandCursor</cursorShape>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>KULLANICILAR</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/resource.qrc">
               <normaloff>:/resim/users.png</normaloff>:/resim/users.png</iconset>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_employees">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>PERSONELLER</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/resource.qrc">
               <normaloff>:/resim/istockphoto-671677466-612x612.jpg</normaloff>:/resim/istockphoto-671677466-612x612.jpg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_customers">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>MÜŞTERİLER</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_vehicles">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>ARAÇLAR</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/resource.qrc">
               <normaloff>:/resim/istockphoto-671677466-612x612.jpg</normaloff>:/resim/istockphoto-671677466-612x612.jpg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_drivers">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>ŞOFÖRLER</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_repairs">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>ARAÇ BAKIM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_contracts">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>SÖZLEŞMELER</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_routes">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>ROTA PLANLAMA</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_trips">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>SEFERLER</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_attendance">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>PUANTAJLAR</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_payments">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>HAKEDİŞLER</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_finance">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>MALİ YÖNETİM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_constants">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>SABİTLER</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_reports">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>RAPORLAR</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_settings">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>AYARLAR</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="content_frame">
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QStackedWidget" name="mainStack">
           <widget class="QWidget" name="page_main"/>
           <widget class="QWidget" name="page_modul_container"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../icons/resource.qrc"/>
 </resources>
 <connections/>
</ui>
